jkube-kit/build/service/docker/src/test/java/org/eclipse/jkube/kit/build/service/docker/helper/ImageNameFormatterTest.java

```diff
@@ -55,7 +52,7 @@ public void invalidFormatChar() {
             formatter.format("bla %z");
             fail();
         });
-        assertThat("Doesnt match", result.getMessage(), containsString("%z"));
+        assertThat(result.getMessage()).as("Doesnt match").contains("%z");
```

Comment:
Maybe better just use the built-in AssertJ assertThat(result).hasMessageContaining("No image name format element '%z' known")